src/pkg/cluster/state.go

```diff
@@ -107,7 +108,8 @@ func (c *Cluster) InitZarfState(ctx context.Context, initOptions types.ZarfInitO
 		// Try to create the zarf namespace.
 		spinner.Updatef("Creating the Zarf namespace")
 		zarfNamespace := NewZarfManagedNamespace(ZarfNamespaceName)
-		if _, err := c.CreateNamespace(ctx, zarfNamespace); err != nil {
+		// NOTE: Should this ignore if namespace exists?
+		_, err := c.Clientset.CoreV1().Namespaces().Create(ctx,zarfNamespace, metav1.CreateOptions{})
```

Comment:
Original implementation skips create if namespace already exists. Is this required here?

Reply:
It should not be required as this only runs if state does not already exist.